pkg/upgrade/upgrade.go

```diff
@@ -55,37 +55,23 @@ const (
 	ZosPackage = "zos.flist"
 )
 
-type RegistrarVersion struct {
-	SafeToUpgrade bool   `json:"safe_to_upgrade"`
-	Version       string `json:"version"`
-}
-
-func getRolloutConfig(ctx context.Context, gw *stubs.RegistrarGatewayStub) (RegistrarVersion, []uint32, error) {
+func getRolloutConfig(ctx context.Context, gw *stubs.RegistrarGatewayStub) (client.ZosVersion, []uint32, error) {
 	env := environment.MustGet()
 	config, err := environment.GetConfig()
 	if err != nil {
-		return RegistrarVersion{}, nil, errors.Wrap(err, "failed to get network config")
+		return client.ZosVersion{}, nil, errors.Wrap(err, "failed to get network config")
 	}
 
 	// if we are on devnet just update we don't need to update the version throught out the registrar
-	v := "v0.0.0"
+	v := client.ZosVersion{Version: "v0.0.0", SafeToUpgrade: true}
```

Comment:
why setting version to zero?

Reply:
this was already the flow, I just updated how it stored